src/main/java/Storage.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Represents a storage that loads from a save file all the previous undeleted task from a 
+ * previous run of <code>V</code> and saves all tasks after shutting down <code>V</code>.
+ */
+public class Storage {
+    private ArrayList<String> commandArray = new ArrayList<String>();
+    private String saveFilePath;
+
+    /**
+     * Clears all data from previous save file to allow overwriting of data.
+     * @param saveFile File object of the save file
+     */
+    public void clearPreviousSave(File saveFile) {
+        try {
+            FileWriter clearSaveFile = new FileWriter(saveFile);
+            clearSaveFile.write("");
+            clearSaveFile.close();
+        } catch (IOException error) {
+            System.out.println(error);
+        }
+    }
+
+    /**
+     * Loads save file using file path saved in <code>saveFilePath</code> attribute.
+     * Stores all the commands as a String in an array.
+     */
+    public void loadSave() {
```

Comment:
the name can be better. reason: does `loadSave` mean load (read from file) or save (write to file)?